### ISSUE TITLE
ModuleNotFoundError: No module named 'CriCodecs'

### ISSUE THREAD

**Issue by Timo654** [LittleChungi/PyCriCodecs#1]
Hi,

I followed the install instructions and tried to run the examples provided, but I got a ModuleNotFoundError.

I tried installing the setup.py in CriCodecs folder, but that also failed for me.

Is there anything I'm missing?

Thanks!

**Comment by Youjose** [comment]
Hello there,

Thanks for noticing the error, this was caused recently because I changed the C-extension CriCodecs name to PyCriCodecs.CriCodecs which caused some problems with imports and whatnot, I reverted back it for now, and will probably keep it that way. I hopefully fixed this error now, so redownload install again.

Try uninstalling the old PyCriCodecs before with `pip uninstall PyCriCodecs` and reinstall. Tell me how it goes.
